test/ZNSAccessController.test.ts

```diff
@@ -46,6 +47,16 @@ describe("ZNSAccessController", () => {
         }, Promise.resolve()
       );
     });
+
+    it("Should revert when passing 0x0 address to assing roles", async () => {
+      await expect(
+        deployAccessController({
+          deployer,
+          governorAddresses: [ ethers.constants.AddressZero ],
+          adminAddresses: [ ethers.constants.AddressZero ],
+        })
+      ).to.be.revertedWith("ZNSAccessController: Can't grant role to zero address");
```

Comment:
Consider adding the new errors to the "errors" file and reference them as constants like we have been doing for all the others